style.css

```diff
@@ -0,0 +1,585 @@
+
+#partners h1 {
+  color: #8f8f91;
+}
+
+#partners hr {
+  margin-bottom: 50px;
+}
+
+#footer-section {
+  display: flex;
+  align-items: center;
+  padding: 5% 17%;
+  gap: 65px;
+  max-width: 90%;
+}
+
+#footer-content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+footer {
+  display: none;
+}
+
+/* -----------------------------------------------------Start  of desktop styling ------------------------------------------------------------------------------------ */
+@media screen and (width >= 768px) {
+  header {
+    height: 27vh;
+  }
+
+  header nav {
+    display: none;
+  }
+
+  #menu {
+    order: 1;
+    display: flex;
+    margin-right: 15%;
+    position: absolute;
+    right: 0;
+  }
+
+  .material > * {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  #hero-section > * {
+    max-width: 65%;
+    margin-left: 10%;
+  }
+
+  #hero-section > .title {
+    font-size: 4rem;
+    background-image: url('./assets/Mobile/hero/wallpaperflare.com_wallpaper.jpg');
+    background-size: cover;
+    /* stylelint-disable */
+    background-clip: text;
+    -webkit-background-clip: text;
+    /* stylelint-enable */
+    color: transparent;
+    margin-right: auto;
+  }
+
+  #hero-section > .title > span {
+    font-size: 2.6rem;
+  }
+
+  .material > p {
+    text-align: center;
+    font-size: 12px;
+    font-weight: 700;
+    line-height: 25px;
+    letter-spacing: 0.72px;
+  }
+
+  #hero-section p {
+    line-height: normal;
+  }
+
+  #hero-section > p:first-of-type {
+    color: #515151;
+    font-size: 25.49px;
+    font-weight: 400;
+    letter-spacing: -0.255px;
+  }
+
+  #hero-section p:nth-of-type(2) {
+    margin-top: 10vh;
+    margin-bottom: auto;
+    font-size: 27.614px;
+    font-weight: 700;
+    letter-spacing: -0.552px;
+  }
+
+  .section-title {
+    font-size: 30px;
+    font-weight: 400;
+    line-height: 28px;
+    letter-spacing: -0.3px;
+  }
+
+  #products-wrapper > .section-title {
+    color: #000;
+  }
+
+  #products-wrapper > hr {
+    margin-bottom: 50px;
+  }
+
+  #materials-wrapper hr {
+    width: 51px;
+    height: 2px;
+  }
+
+  .materials-list {
+    display: flex;
+    flex-direction: row;
+    margin: 0 7%;
+  }
+
+  .material {
+    grid-template-rows: 1fr 1fr 1fr;
+    grid-template-columns: auto;
+    padding: 12px;
+  }
+
+  .material > img {
+    width: 30%;
+    height: auto;
+    justify-self: center;
+  }
+
+  .material > h3 {
+    font-size: 24px;
+    font-weight: 700;
+    line-height: 28px;
+    letter-spacing: -0.24px;
+  }
+
+  #materials-wrapper {
+    padding: 4% 2%;
+    height: auto;
+  }
+
+  #materials-wrapper > .cta {
+    display: none;
+  }
+
+  #materials-wrapper > a {
+    display: flex;
+    margin-top: 2%;
+  }
+
+  #products-list {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    padding: 0 15%;
+  }
+
+  .product {
+    grid-template-rows: auto;
+    transition: all;
+    transition-duration: 1s;
+  }
```

Comment:
- [ ] Also please make sure your design for the cards in the "Crafts" section matches the Behance design by including:
  - the proper font size, and text color.
  - also please make sure to add the gray-pattern image behind each craft image as shown in the Behance design.
- Behnace design:
  ![image](https://github.com/Estete9/IliArtes/assets/104968236/7c9085c7-0425-4382-8bb5-f253ec459f7b)
- Your design:
  ![image](https://github.com/Estete9/IliArtes/assets/104968236/5a4a56c9-df73-4e2c-a844-3970cf806378)
- [ ] In the mobile version please make sure only the first 2 cards are displayed and when the more button is clicked the rest should appear as currently all the cards are being displayed.
- Behance design:
  ![image](https://github.com/Estete9/IliArtes/assets/104968236/7e087a20-6f11-4d2a-9412-204bd456a454)